src/harness/testcases/grant_testcase.py

```diff
@@ -133,6 +133,102 @@ def test_10_7_4_1_3_1_2_1(self):
     self.assertFalse('grantId' in response)
     self.assertEqual(response['response']['responseCode'], 103)
 
+ @winnforum_testcase
+  def test_WINFF_FT_S_GRA_2(self):
+    """Successful CBSD grant request.
+
+    Incumbent is present in the GAA frequency range requested by the
+    CBSD which is outside the protection zone..
+    """
+    # Register the device
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    request = {'registrationRequest': [device_a]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+    # Check registration response
+    self.assertEqual(response['response']['responseCode'], 0)
+    cbsd_id = response['cbsdId']
+    del request, response
+    # Create and trigger the ESC Zone
+    esc_zone_not_contain_device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'esc_zone_not_contain_device_a.json')))
+    zoneMessage = {'zone':esc_zone_not_contain_device_a}
```

Comment:
Please use `request` or `zone_request` to remain consistent with our variable naming convention.